datasets/conll2000/README.md

```diff
@@ -3,6 +3,98 @@ language:
 - en
 paperswithcode_id: conll-2000-1
 pretty_name: CoNLL-2000
+dataset_infos:
+- config_name: conll2000
```

Comment:
do we need the (non-default) config_name for backward compatibility?

Reply:
The dataset explicitly defines one configuration with this name, but since there's only one there's no ambiguity. Let me tweak the YAML loading a bit and we can remove this line :)

Reply:
I removed config name for conll2000 in the YAML part, and for everything to connect smoothly I had to remove the definition of the config in the dataset script itself. I'll see if it's something I can do for all the datasets that have one configuration